### PR TITLE
Fix for new value of GenerateDebugInformation in VS2015.

### PR DESCRIPTION
**Pull request by g012**
PR for https://github.com/premake/premake-core/issues/483

### CODE REVIEW

File: src/actions/vstudio/vs2010_vcxproj.lua [hunk 1]
```diff
@@ -1304,7 +1304,10 @@
 
 
 	function m.generateDebugInformation(cfg)
-		m.element("GenerateDebugInformation", nil, tostring(cfg.flags.Symbols ~= nil))
+		local d = cfg.flags.Symbols ~= nil
+		local v
+		if _ACTION >= "vs2015" then v = d and "Debug" or "No" else v = tostring(d) end
```
Comment (starkos) [inline src/actions/vstudio/vs2010_vcxproj.lua]: A nitpick: but the coding convention is to put then/else on their own lines, and it does make the code easier to follow.

**Comment by starkos** [comment]
Can we get unit tests for the new output values?

**Comment by g012** [comment]
Sure.
I added a vc2015 folder, I don't know if that's what you want or if you prefer to have two files for symbols in vc2010.

**Comment by tvandijck** [comment]
I like it this way... LGTM...

**Comment by samsinsane** [comment]
I don't think having a vc2015 folder is a good idea, it separates the tests for the same function into two files unnecessarily. In my opinion, the tests are for the functions, not for the versions of VS. The `test_symbols.lua` file should contain all the tests for VS2010-2013 and VS2015. But that's just my opinion on this, I don't know exactly what the plan was for the VS tests.

**Comment by starkos** [comment]
FWIW, I agree with Sam on this one: it would be good to have all of those tests in the same place.

**Comment by tvandijck** [comment]
https://github.com/premake/premake-core/pull/525

might supersede this PR.. Although adding the tests would still be welcome...

**Comment by tvandijck** [comment]
I think this was all fixed in 525... I'm closing this one, feel free to reopen if I'm wrong, or missed something. As it stands it has conflicts and can't be merged, so an update would be needed anyway.

**Pull request closed without merging**